src/js/components/MaskedInput/maskedinput.stories.js

```diff
@@ -198,8 +207,111 @@ class IPv4MaskedInput extends Component {
   }
 }
 
+const DropContent = props => {
+  const { date, onSelect, time, onClose, onChange } = props;
+  return (
+    <Box align="center">
+      <Calendar date={date} onSelect={onSelect} />
+      <Box width="small" align="center" margin={{ bottom: 'small' }}>
+        <MaskedInput
+          mask={[
+            {
+              length: [1, 2],
+              options: [
+                '1',
+                '2',
+                '3',
+                '4',
+                '5',
+                '6',
+                '7',
+                '8',
+                '9',
+                '10',
+                '11',
+                '12',
+              ],
+              regexp: /^1[1-2]$|^[0-9]$/,
+              placeholder: 'hh',
+            },
+            { fixed: ':' },
+            {
+              length: 2,
+              options: ['00', '15', '30', '45'],
+              regexp: /^[0-5][0-9]$|^[0-9]$/,
+              placeholder: 'mm',
+            },
+            { fixed: ' ' },
+            {
+              length: 2,
+              options: ['am', 'pm'],
+              regexp: /^[ap]m$|^[AP]M$|^[aApP]$/,
+              placeholder: 'ap',
+            },
+          ]}
+          value={time}
+          name="maskedInput"
+          onChange={onChange}
+        />
+      </Box>
+      <Box margin={{ bottom: 'small' }}>
```

Comment:
If `margin` is the reason we are using this Box, I think we can remove the Box and move `margin` internally to Button itself.

Reply:
Sure. Pushed up changes.